ui/src/main/webapp/tsconfig-tsModels.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "module": "system",
+    "moduleResolution": "node",
+    "sourceMap": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "removeComments": false,
+    "noImplicitAny": false,
+    "rootDirs": [".", "../../src/main/webapp/"],
+    "outDir": "../windup-web/app/tsModels",
+    "sourceRoot": "/windup-web/"
+  },
+  "exclude": [
+    "../../src/main/webapp/**"
+  ]
+}
```

Comment:
This file is already deleted - please remove the first commit (2ab5d4afb3724110e4beb940b0f16162d1b989cf)